charts/images.yaml

```diff
@@ -59,8 +59,8 @@ images:
   tag: v0.5.0
 - name: csi-plugin-alicloud
   sourceRepository: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver
-  repository: registry.eu-central-1.aliyuncs.com/acs/csi-plugin
-  tag: v1.20.5-ff6490f-aliyun
+  repository: registry.cn-hangzhou.aliyuncs.com/acs/csi-plugin
```

Comment:
```suggestion
  repository: registry.eu-central-1.aliyuncs.com/acs/csi-plugin
```

Reply:
We would suggest to use images located other than Mainland China by default. You may use image-overwrite to change images when you deploy it.